components/common/WalletModal.vue

```diff
@@ -291,8 +300,21 @@ export default class WalletModal extends mixins(UseApiMixin, ChainMixin) {
     .modal-card-head {
       background: unset;
       border-bottom: 1px solid #000000;
-      padding: 8px 32px;
+      padding: 20px 0px;
+      display: flex;
 
+      .header-btn {
+        padding: 10px !important;
+        border-radius: 4px;
+        position: absolute;
+      }
+      .delete {
+        right: 14px;
+      }
```

Comment:
I see a duplicate class name `.delete` here.

Reply:
You are quite right, pushing the fix RN